pkg/controller/plan/adapter/ovirt/builder.go

```diff
@@ -702,3 +663,233 @@ func (r *Builder) LunPersistentVolumeClaims(vmRef ref.Ref) (pvcs []core.Persiste
 	}
 	return
 }
+
+func (r *Builder) SupportsVolumePopulators() bool {
+	return true
```

Comment:
the way it is used in kubevirt.go, it should return true only for cold migrations to the local cluster